Jenkinsfile

```diff
@@ -7,8 +7,8 @@ pipeline {
      stages {
          stage('Build') {
              steps {
-                 sh 'mvn -B -DskipTests clean install'
+                 sh 'mvn -B clean install -Pintegration'
```

Comment:
I am not sure if we want this. This changes Jenkins file for everyone, so any time someone tries to build the app, it force runs integration tests. Tests should be run in parallel in stage different than build. I would create parallel stages for tests and each would run "one type". This will cut down on time needed to run tests and will give us more outputs.

Reply:
Okay, let's do this in a separate task for adjusting the Jenkinsfile.